### PR TITLE
Feature/1 create readme

### LINKED ISSUE #1: This is a demo - we need to add a README.md
I'd like to have a README.md which includes directions for installing dependencies.

### PR DESCRIPTION
**Pull request by KranzTech**
Fixes #1 No Readme file
There was no readme file for this repo.  This fixes that issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is a readme that will show up at the bottom of the main page.
+
+Please include any information that would be helpful for people to use the code above.
```
Comment (nightflame137) [inline README.md:3]: You might add more directions to this file to increase user friendliness and understanding.

**Review by nightflame137 — CHANGES_REQUESTED** [review]
It looks good except for the one line change I requested

File: README.md (hunk 1 above)
Comment (KranzTech) [reply README.md:3]: Thanks for the great suggestion.  I updated the file.

**Review by nightflame137 — APPROVED** [review]
Looks good!

**Pull request merged**